rst_files/types_methods.rst

```diff
@@ -495,14 +447,13 @@ If the interpreter can't find a match in immediate parents (supertypes) it proce
 
 * If not, we get an error
 
-This is the process that leads to the following error 
+This is the process that leads to the following error
```

Comment:
Doesn't yield the desired error, because this block inherits `+(x::Integer, y::String)` from an earlier code block. Will eventually need to replace with a valid example.